ReactAndroid/src/main/java/com/facebook/react/bridge/ReactMarkerConstants.java

```diff
@@ -59,8 +59,8 @@
   UNPACKING_JS_BUNDLE_LOADER_CHECK_END,
   UNPACKING_JS_BUNDLE_LOADER_EXTRACTED,
   UNPACKING_JS_BUNDLE_LOADER_BLOCKED,
-  loadApplicationScript_startStringConvert,
-  loadApplicationScript_endStringConvert,
+  loadBundle_startStringConvert,
```

Comment:
This is the only thing that I'm a little bit concerned about - renaming our perf markers could be a breaking change for our perf instrumentation. I think it might be better to keep these the same, even though they won't match the function name anymore.

Reply:
Totally valid concert, we'll revert that change.